stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -401,17 +385,6 @@ internal constructor(
         this.errorHandlers = errorHandlers.sorted()
     }
 
-    private fun logPlugins(plugins: List<Any>) {
-        if (plugins.isNotEmpty()) {
-            val jointString = plugins.joinToString { plugin ->
-                plugin::class.qualifiedName ?: "plugin without qualified name"
-            }
-            logger.d { "Plugins found: $jointString" }
-        } else {
-            logger.d { "No plugins found for this request." }
-        }
-    }
```

Comment:
As far as all of our Plugins implement all methods, this logger method does not make sense anymore